--- thunderbolt-software-user-space-2017.01.19/fwupdate/libtbtfwu/CMakeLists.txt.orig	2017-01-19 15:35:27.000000000 +0100
+++ thunderbolt-software-user-space-2017.01.19/fwupdate/libtbtfwu/CMakeLists.txt	2017-03-13 21:39:05.651271759 +0100
@@ -73,7 +73,7 @@
 add_library(tbt_static STATIC ${LIB_SRC_FILES})
 set_target_properties(tbt_static PROPERTIES OUTPUT_NAME tbtfwu)
 
-target_link_libraries(tbtfwu ${LIBDBUS-C++-1_LIBRARIES})
+target_link_libraries(tbtfwu ${LIBDBUS-C++-1_LIBRARIES} pthread)
 
 set_target_properties(tbtfwu
     PROPERTIES LINK_DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/visibility.map)
